cmd/egress/cmd.go

```diff
@@ -42,6 +42,14 @@ func NewCmdValidateEgress() *cobra.Command {
 
 	validateEgressCmd := &cobra.Command{
 		Use: "egress",
+		Short: "Verify essential openshift domains are reachable from given subnet ID.",
+		Long:  `Verify essential openshift domains are reachable from given subnet ID.`,
+		Example: `For AWS, ensure your credential environment vars 
```

Comment:
nit: we should consider making validateEgressCmd cloud-agnostic
creds can be both gcp or aws although our current form is not like that in cmd (but we support it in pkg/)

Reply:
created card: https://issues.redhat.com/browse/OSD-11384